product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx

```diff
@@ -33,6 +33,20 @@ CREATE TABLE dept_uk2 (
 
 -   Make sure table definitions are well established before creating publications. Unless you use the DDL change replication feature, as described in [Replicating DDL changes](../../07_common_operations/08_replicating_ddl_changes/#replicating_ddl_changes), if a table definition changes, you must delete and re-create any publication containing the table along with its associated subscription. Otherwise replication might fail. The same applies for the table definitions in a primary definition node and its associated primary nodes. You can see replication failures in the replication history.
 -   Views can be members of snapshot-only publications. In the subscription database, a view is replicated as a table.
+-   Materialized views can be members of snapshot-only publications from Oracle to EDB Postgres Advanced Server databases. The materialized views are replicated as tables. 
+    1. Use the `‘tables’` parameter for specifying materialized views while creating a publication via EDB Replication Server CLI. While creating a snapshot-only publication via Replication Console, the view type currently appears as a table. 
+    1. We recommend that useres create the views with auto-refresh in Oracle. If the views are not auto-refresh they are marked as invalid upon updating the base table until refreshed explicitly. EDB Replication Server currently does not support subscription creation or snapshot for invalid views.
+    
+       The following error appears during a snapshot where the view is invalid:
+
+       ```sql
+       INFO: MTK-14001: One or more tables are missing from the source Oracle database.
+       ```
+
+       With this error, the user must refresh the view before taking the snapshot. 
+
+       !!! Note
+           Snapshot-only replication for views is currently tested and certified only for Oracle to EDB Postgres Advanced Server databases.
```

Comment:
Change to Snapshot-only replication for **Materialized** views.
Views are supported for all combinations. 
Materialized views are tested and certified only for Oracle to EDB Postgres Advanced Served databases.